src/Cosi-BLS/block.lisp

```diff
@@ -185,7 +185,6 @@ added to the blockchain."
             (cosi/proofs:public-key-to-address public-key)))
          (transactions (list genesis-transaction))
          (block (create-block nil nil nil nil transactions)))
-    ;(cosi-simgen:node-dispatcher :genesis-block :blk block)
```

Comment:
(see comment in previous commit that commented this code out: I am not convinced this code should be eliminated, and now the doc is out of sync)